src/NUnitFramework/framework/Compatibility/System.Collections/CollectionDebuggerView.cs

```diff
@@ -7,7 +7,7 @@
 // Authors:
 //	Marek Safar  <marek.safar@gmail.com>
 //
-// Copyright (C) 2009 Novell, Inc (http://www.novell.com)
```

Comment:
D'ya think the two links in copyright notices should be left as is? Otherwise everything else in this PR makes me very happy! 😄

Reply:
I don't know why not, but I can change if there's a concern. Both forms redirect to their new domain name.

Reply:
I'm not too worried, I'd just normally err on the side of safety myself and not touch copyright notices - as I don't really understand the details! I'll approve, and leave it with you to decide whether to merge as is, or make the change. 🙂

Reply:
If it's up to me, I'm going to act as though this is fine because I don't want to be in a world where it isn't. I guess we'll find out if that's realistic. 😃